include/sys/syscall_lookup.h

```diff
@@ -380,3 +380,26 @@ SYSCALL_LOOKUP(munmap,                     2)
 #ifdef CONFIG_CRYPTO_RANDOM_POOL
   SYSCALL_LOOKUP(arc4random_buf,           2)
 #endif
+
+SYSCALL_LOOKUP(nxmutex_init,               1)
```

Comment:
function which doesn't touch kernel space data or function should move to libs/libc/mutex. nxmutex_init is one of example.